cmd/boulder-va/main.go

```diff
@@ -32,7 +32,7 @@ type RemoteVAGRPCClientConfig struct {
 	// of RVAs deployed in the same datacenter.
 	//
 	// TODO(#7615): Make mandatory.
-	Perspective string `validate:"omitempty"`
+	Perspective string `validate:"required"`
```

Comment:
I think this might have accidentally gotten carried over in a merge/rebase. We're not yet ready to make this `required`.